### ISSUE TITLE
As an Employer, I would like to know that my upload was successfully uploaded.

### ISSUE THREAD

**Issue by oghaffari** [18F/dol-whd-14c#171]
Receive a confirmation message or something similar.

**Comment by binwang89** [comment]
Tested this user story and found below issues:

1. file type display is not match the requirements( For milestone 1, supported file types do not include doc, docx, xls.)
![image](https://user-images.githubusercontent.com/25330625/34424637-1d01ab90-ebf3-11e7-972d-5fccc2d1f80d.png)
 
2. Error message/string needed for invalid type and/or size.
I uploaded a excel file, and nothing happy. The system did not respond any error message about invalid file type. For other file types (PDF, docx, Image file), I could review the file after i uploaded it.  The system should not accept any file except for PDF file type. 
![image](https://user-images.githubusercontent.com/25330625/34424722-a3d0f126-ebf3-11e7-8304-668dc93c3601.png)
